pinot-common/src/main/java/org/apache/pinot/common/config/provider/TableCache.java

```diff
@@ -390,6 +514,22 @@ private List<Schema> getSchemas() {
     return schemas;
   }
 
+  public boolean isLogicalTable(String logicalTableName) {
```

Comment:
Can this function use `getActualLogicalTableName` and check for null ?

Reply:
The current implementation looks alright. It is checking within the map.

```return _logicalTableInfoMap.containsKey(logicalTableName);```